ResourceManager.cpp

```diff
@@ -1,42 +1,41 @@
 #include "ResourceManager.h"
 #include "Archives/VolFile.h"
 #include "Archives/ClmFile.h"
-//#include "Streams/StreamReader.h"
+#include "Streams/SeekableStreamReader.h"
 #include "XFile.h"
 
-using namespace std;
 using namespace Archives;
 
-ResourceManager::ResourceManager(const string& archiveDirectory)
+ResourceManager::ResourceManager(const std::string& archiveDirectory)
 {
-	vector<string> volFilenames = XFile::GetFilesFromDirectory(archiveDirectory, ".vol");
+	std::vector<std::string> volFilenames = XFile::GetFilesFromDirectory(archiveDirectory, ".vol");
 
-	for (const string& volFilename : volFilenames) {
-		ArchiveFiles.push_back(make_unique<VolFile>(volFilename.c_str()));
+	for (const auto& volFilename : volFilenames) {
+		ArchiveFiles.push_back(std::make_unique<VolFile>(volFilename.c_str()));
 	}
 
-	vector<string> clmFilenames = XFile::GetFilesFromDirectory(archiveDirectory, ".clm");
+	std::vector<std::string> clmFilenames = XFile::GetFilesFromDirectory(archiveDirectory, ".clm");
 
-	for (const string& clmFilename : clmFilenames) {
-		ArchiveFiles.push_back(make_unique<ClmFile>(clmFilename.c_str()));
+	for (const auto& clmFilename : clmFilenames) {
+		ArchiveFiles.push_back(std::make_unique<ClmFile>(clmFilename.c_str()));
 	}
 }
 
 // First searches for resources loosely in provided directory. 
 // Then, if accessArhives = true, searches the preloaded archives for the resource.
-unique_ptr<SeekableStreamReader> ResourceManager::GetResourceStream(const string& filename, bool accessArchives)
+std::unique_ptr<SeekableStreamReader> ResourceManager::GetResourceStream(const std::string& filename, bool accessArchives)
 {
 	if (XFile::PathExists(filename)) {
-		return make_unique<FileStreamReader>(filename);
+		return std::make_unique<FileStreamReader>(filename);
 	}
 
 	if (!accessArchives) {
 		return nullptr;
 	}
 
-	for (unique_ptr<ArchiveFile>& archiveFile : ArchiveFiles)
+	for (auto& archiveFile : ArchiveFiles)
 	{
-		string internalArchiveFilename = XFile::GetFilename(filename);
+		std::string internalArchiveFilename = XFile::GetFilename(filename);
```

Comment:
I suppose this could be `auto` as well, though I don't think it matters which way we go for simple cases like this.

Reply:
I don't know that I am ready to pepper the code with this many autos yet.